runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -1015,6 +1064,28 @@ TR_J9ServerVM::sameClassLoaders(TR_OpaqueClassBlock * class1, TR_OpaqueClassBloc
 bool
 TR_J9ServerVM::isUnloadAssumptionRequired(TR_OpaqueClassBlock *clazz, TR_ResolvedMethod *methodBeingCompiled)
    {
+   TR_OpaqueClassBlock * classOfMethod = static_cast<TR_ResolvedJ9JITaaSServerMethod *>(methodBeingCompiled)->classOfMethod();
+   if (clazz == classOfMethod)
+      return false;
+
+      {
+      OMR::CriticalSection getRemoteROMClass(_compInfoPT->getClientData()->getROMMapMonitor());
+      auto it_class = _compInfoPT->getClientData()->getROMClassMap().find((J9Class*) clazz);
+      if (it_class != _compInfoPT->getClientData()->getROMClassMap().end())
+         {
+         if (!isAnonymousClass(it_class->second.romClass))
+            {
+            if (it_class->second.classLoader == getSystemClassLoader())
+               return false;
+            auto it_classOfMethod = _compInfoPT->getClientData()->getROMClassMap().find((J9Class*) classOfMethod);
+            if (it_classOfMethod != _compInfoPT->getClientData()->getROMClassMap().end())
+               return (it_class->second.classLoader != it_classOfMethod->second.classLoader);
+            }
+         else
+            return true;
```

Comment:
Please use curly brackets around `return true`. The convention is that the `if` and the `else` blocks have to look the same: either both have curly brackets or none of them has.